hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/ratis/TestOzoneManagerRatisServer.java

```diff
@@ -205,6 +205,7 @@ public void verifyRaftGroupIdGenerationWithCustomOmServiceId() throws
         .setOMServiceId(customOmServiceId)
         .build();
     // Starts a single node Ratis server
+    omRatisServer.stop();
     OzoneManagerRatisServer newOmRatisServer = OzoneManagerRatisServer
```

Comment:
@ChenSammi omRatisServer is started at the begin of each test case and stop after that. However in this new test case, it started a different OzoneManagerRatisServer named newOmRatisServer, it has to be stopped explicitly to avoid leak. Also, the omRatisServer has to be stopped to avoid metric registration failure.